queries/orderbook/batch_rewards.sql

```diff
@@ -28,13 +28,136 @@ WITH observed_settlements AS (SELECT
                                WHERE block_deadline >= {{start_block}}
                                  AND block_deadline <= {{end_block}}
                                GROUP BY ss.auction_id),
+-- protocol fees:
+order_surplus AS (
+    SELECT
+        ss.winner as solver,
+        at.auction_id,
+        s.tx_hash,
+        t.order_uid,
+        o.sell_token, -- the total amount the user sends
+        o.buy_token,
+        t.sell_amount,
+        t.buy_amount,
+        oe.surplus_fee as observed_fee,
+        o.kind,
+        CASE
+            WHEN o.kind = 'sell'
+                THEN t.buy_amount - o.buy_amount / (o.sell_amount + o.fee_amount) * t.sell_amount
```

Comment:
```suggestion
                THEN t.buy_amount - t.sell_amount * o.buy_amount / (o.sell_amount + o.fee_amount)
```

So that it is easier to parse?